eos-payg-csv/eos-payg-csv

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+import csv
+import os
+import shutil
+import subprocess
+import sys
+
+PERIODS = "1d 2d 3d 4d 5d 30d 60d 90d 365d infinite".split()
+TMPDIR = "tmp"
+OUTDIR = "codes"
+
+def period_to_display_str(period):
+    if period == "infinite":
+        return "infinite"
+
+    days, _ = period.split("d")
+
+    if days == "1":
+        return "1 day"
+
+    return "{0} days".format(days)
+
+def key_to_tmp_filename(key):
+    return os.path.join(TMPDIR, "{}.key".format(key))
+
+def rm_and_mkdir(dirname):
+    if os.path.isdir(dirname) and not os.path.islink(dirname):
+        if len(os.listdir(dirname)) == 0:
+            os.rmdir(dirname)
+        else:
+            shutil.rmtree(dirname)
+    os.mkdir(dirname)
```

Comment:
In the case that `os.path.islink(dirname)` is true, this `mkdir()` will still be called, and will fail (I think). Is that desired?

Why are links handled specially in this function? Could do with a comment.